src/examples/java/com/nylas/examples/other/RoomResourceExample.java

```diff
@@ -0,0 +1,23 @@
+package com.nylas.examples.other;
+
+import com.nylas.NylasAccount;
+import com.nylas.NylasClient;
+import com.nylas.RoomResource;
+import com.nylas.RoomResources;
+import com.nylas.examples.ExampleConf;
+
+import java.util.List;
+
+public class RoomResourceExample {
+    public static void main(String[] args) throws Exception {
```

Comment:
currently the java sdk is using tabs rather than spaces.  I don't have a strong preference between them, but do think it's good to stay consistent if we can.

Reply:
I'm still seeing spaces here.  Is it just me?

Reply:
I'm not sure, I just committed it again it should be fixed now: 487ec1a3a33cf72a5bd72677b8131ad755d0cc37